src/main.dev.js

```diff
@@ -581,3 +581,8 @@ app.on('activate', () => {
   // dock icon is clicked and there are no other windows open.
   if (mainWindow === null) createWindow();
 });
+
+ipcMain.on('reload', () => {
+  app.quit();
+  createWindow();
+});
```

Comment:
I'm getting errors with this on Windows. I did a bit of research and this can probably be changed to:

```
ipcMain.on('reload', () => {
  app.relaunch();
  app.exit();
});
```

Reference: https://stackoverflow.com/a/60186387

Reply:
Definitely doesn't work on Linux. That was the first thing I tried before asking you.

Reply:
Did you package the application and run it from the production build? It doesn't work on the dev build for me.

Reply:
It worked for me in dev (yarn start), but I wasn't able to package it (yarn package)
```
electron-builder  version=22.10.5 os=5.15.10-arch1-1
  • loaded configuration  file=package.json ("build" field)
  • writing effective config  file=release/builder-effective-config.yaml
  • packaging       platform=linux arch=x64 electron=13.3.0 appOutDir=release/linux-unpacked
  • Unpacking electron zip  zipPath=undefined
  ⨯ editions-autoloader-none-broadened: Unable to determine a suitable edition, even after broadening.  stackTrace=
                                                                                                          Error: editions-autoloader-none-broadened: Unable to determine a suitable edition, even after broadening.
```

Reply:
Can you try `yarn package --linux`?

Reply:
The exact same error occurs.
If you want the full output I put it on [Pastebin](https://pastebin.com/MeiSt34w)

Reply:
Try with the `Sonixd(dev)_2021-12-21` build.

Reply:
Sorry, I don't know what you want me to do. Where do I find the build?

Reply:
The build is here: http://ftp.sonixd.com/

`Sonixd(dev)_2021-12-21T00:41:17_b9f2f6d814b3ce2d562585a0ba8ad06cf3d2b836` uses the code from the stackoverflow. 

`Sonixd(dev)_2021-12-21T00:26:23_e581aedbcc985e9d7fd30ba8d9c328daa63e9f6f` uses your implementation.

Reply:
The Stackoverflow build does not restart. It just closes.
The build using my implementation works on my system.

Reply:
Do you have discord? Might be easier to discuss in chat.
https://discord.gg/PcJCMgthGa

Reply:
No sorry, I don't use discord. You can message me on [Matrix/Element](https://element.io/get-started) though!
You can use it from the web if you don't want to install it and directly sign in using GitHub.